x/lockup/spec/02_state.md

```diff
@@ -0,0 +1,49 @@
+<!--
+order: 2
+-->
+
+# State
+
+## Locked coins management
+
+Locked coins are all stored in module account for `lockup` module which is called `LockPool`.
+When user lock coins within `lockup` module, it's moved from user account to `LockPool` and a record (`PeriodLock` struct) is created.
+
+Once the period is over, user can withdraw it at anytime from `LockPool`.
+Note:
+- Do we need automate withdraw from the pool?
+- When querying for full lock amount, unlocked amount but which is in lock pool yet should be queried?
+- Is it needed to withdraw by record or by coins? (Or even withdraw full for unlocked?)
```

Comment:
I think either:
"Unlock by PeriodLock ID" or "Unlock all Unlockable Coins"

Reply:
I think Unlock all Unlocakable Coins would be better for user action simplicity.

Reply:
Agreed